src/MongoDB.Analyzer/Core/Linq/LinqExpressionProcessor.cs

```diff
@@ -144,6 +168,45 @@ public static ExpressionsAnalysis ProcessSemanticModel(MongoAnalyzerContext cont
     private static ExpressionSyntax GetNextNestedInvocation(ExpressionSyntax expressionSyntax) =>
         ((expressionSyntax as InvocationExpressionSyntax)?.Expression as MemberAccessExpressionSyntax)?.Expression;
 
+    private static bool PreanalyzeLinqQuerySyntax(SyntaxNode linqExpressionNode, SemanticModel semanticModel, List<InvalidExpressionAnalysisNode> invalidLinqExpressionNodes)
```

Comment:
No way to combine with `PreanalyzeLinqExpression` ? Seems that they are almost the same.